proto/dwtools/abase/l1/StrBasic.s

```diff
@@ -5103,8 +5103,40 @@ strLinesNearest_body.defaults =
 
 let strLinesNearest = _.routineFromPreAndBody( strLinesNearest_pre, strLinesNearest_body );
 
+
 //
 
+/**
+ * The method strLinesNearestReport returns a report about found string in a text within the range { -o.charsRangeLeft- } surrounded by the amount of lines { -o.numberOfLines- } from the source string { -o.src- }.
```

Comment:
Please, try to use simple sentences. 

Also, use `routine` instead of `method`.